src/renderer/i18n/en/deposit.ts

```diff
@@ -15,13 +15,25 @@ const deposit: DepositMessages = {
   'deposit.totalEarnings': 'Your total earnings from the pool',
   'deposit.add.asym': 'Add {asset}',
   'deposit.add.sym': 'Add {assetA} + {assetB}',
+  'deposit.add.state.healthCheck': 'Health check',
+  'deposit.add.state.sending': 'Send deposit transaction',
+  'deposit.add.state.checkResults': 'Check deposit result',
   'deposit.add.state.pending': 'Adding liquidity',
   'deposit.add.state.success': 'Successful deposit',
   'deposit.add.state.error': 'Deposit error',
   'deposit.add.error.chainFeeNotCovered': 'Needed fee of {fee} is not covered by your balance: {balance}',
   'deposit.add.error.nobalances': 'No balances',
   'deposit.add.error.nobalance1': "You don't have any balance of {asset} in your wallet to deposit.",
   'deposit.add.error.nobalance2': "You don't have any balances of {asset1} and {asset2} in your wallet to deposit.",
+  'deposit.interact.bond.sending': 'Send bond transaction',
+  'deposit.interact.bond.checkResults': 'Check bond result',
+  'deposit.interact.unbond.sending': 'Send unbond transaction',
+  'deposit.interact.unbond.checkResults': 'Check unbond result',
+  'deposit.interact.leave.sending': 'Send leave transaction',
+  'deposit.interact.leave.checkResults': 'Check leave result',
+  'deposit.bond.state.error': 'Bond error',
+  'deposit.unbond.state.error': 'Unbond error',
+  'deposit.leave.state.error': 'Leave error',
```

Comment:
To much details ^, let's use following (same) messages for transactions everywhere (deposit, swap, interact) :

`EN`

```
  'common.tx.healthCheck': 'Health check',
  'common.tx.sending': 'Send transaction',
  ''common.tx.checkResult': 'Check result',
```

`DE`
```
  'common.tx.healthCheck': 'Systemcheck',
  'common.tx.sending': 'Sende Transaktion',
  ''common.tx.checkResult': 'Überprüfe Ergebnis',
```